lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -54,6 +60,9 @@ class MysqliStatement implements IteratorAggregate, Statement
     /** @var string */
     protected $types;
 
+    /** @var array<string, array<int, int>> maps parameter names to their placeholder number(s). */
+    protected $placeholderNamesToNumbers = [];
```

Comment:
Let's go private by default, regardless of other properties
```suggestion
    private $placeholderNamesToNumbers = [];
```